api/RTCRtpStreamStats.json

```diff
@@ -0,0 +1,730 @@
+{
+  "api": {
+    "RTCRtpStreamStats": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpStreamStats",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "63"
```

Comment:
Renamed in Firefox 63 per [bug 1480498](https://bugzilla.mozilla.org/show_bug.cgi?id=1480498): https://hg.mozilla.org/mozilla-central/rev/6f781af29708